src/snowflake/cli/_plugins/project/commands.py

```diff
@@ -101,14 +114,51 @@ def dry_run(
     return SingleQueryResult(result)
 
 
+def _add_version_to_project(
+    pm: ProjectManager,
+    project: ProjectEntityModel,
+    prune: bool = False,
+    from_stage: Optional[str] = None,
+    alias: Optional[str] = None,
+    comment: Optional[str] = None,
+):
+    """
+    Adds a version to project. If [from_stage] is not defined,
+    uploads local files to the stage defined in project definition.
+    """
+
+    if not from_stage:
+        cli_context = get_cli_context()
+        from_stage = project.stage
+        with cli_console.phase("Uploading artifacts"):
+            sync_artifacts_with_stage(
+                project_paths=ProjectPaths(project_root=cli_context.project_root),
+                stage_root=from_stage,
+                artifacts=project.artifacts,
+                prune=prune,
+            )
+
+    with cli_console.phase(f"Creating project version from stage {from_stage}"):
+        return pm.add_version(
+            project_name=project.fqn,
+            from_stage=from_stage,
+            alias=alias,
+            comment=comment,
+        )
+
+
 @app.command(requires_connection=True)
 @with_project_definition()
-def create_version(
+def create(
     entity_id: str = entity_argument("project"),
+    no_version: bool = no_version_flag(
+        mutually_exclusive=["prune"],
+    ),
+    prune: bool = PruneOption(mutually_exclusive=["no_version"]),
     **options,
 ):
     """
-    Upload local files and create a new version of a project using those files. If the stage does not exist, it will be created.
+    Creates a project in snowflake and initializes it with a new version created from local files.
```

Comment:
```suggestion
    Uploads local files and creates a project.
```
I'm also fine with the original wording as long as we skip the part about creating versions since it's configurable

Reply:
Changed to
```
    Creates a project in snowflake.
    By default, the project is initialised with a new version created from local files.
```